data/topics.json

```diff
@@ -26,37 +30,52 @@
   },
 
   {
-    "name": "WebRTC issues",
-    "description": "Committed to fixing WebRTC bugs and performance issues",
+    "name": "Native or programmable custom protocol handlers",
+    "description": "Vendors who either support Distributed Web Schemes natively, or allow a browser extension to register itself as the handler for new protocols.",
+    "doc": "programmable-custom-protocol-handlers",
     "beaker": {
-      "level": 0
+      "level": 2,
+      "details": "Built in support for `dat://` and `hyper://` URIs"
     },
     "brave": {
-      "level": 0
+      "level": 2,
+      "details": "Work in progress on `ipfs://` and `ipns://` URI backed by either a public gateway or embedded go-ipfs support: [brave-browser/issues/10220](https://github.com/brave/brave-browser/issues/10220)"
+    },
+    "opera": {
+      "level": 2,
+      "details": "*Opera for Android* supports IPFS with a native handler for `ipfs://` URI that redirects to the HTTP gateway of your choosing. Read more: [IPFS in Opera for Android](https://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/)."
     },
     "chrome": {
-      "level": 0
+      "level": 1,
+      "details": "*Chrome 86* [safelisted](https://blog.chromium.org/2020/09/chrome-86-improved-focus-highlighting.html) Distributed Web Schemes for redirect-based `registerProtocolHandler()`; [Protocol Labs](https://protocol.ai/) and [Igalia](https://igalia.com) work on improving protocol handler APIs in Chromium for both Web and WebExtension contexts."
     },
     "edge": {
-      "level": 0
+      "level": 1,
+      "details": "Supportive, looking into protocol handlers as part of their PWA work"
     },
     "firefox": {
       "level": 1,
-      "details": "Supportive"
+      "details": "Supportive: [bug #1271553](https://bugzilla.mozilla.org/show_bug.cgi?id=1271553)<br/>Prototyping API as a part of [libdweb](https://github.com/mozilla/libdweb): [Protocol handler API](https://github.com/mozilla/libdweb/issues/2)"
```

Comment:
"Supportive" is questionable. Libdweb has no signal of support in some time now, and no action really on protocol handler part.

Reply:
Good point. I've updated the score and rewrote Firefox details – mind taking a quick look?

- c9c4f76 
- 5b6c190

Reply:
👍🏼